docs/spec_draft.md

```diff
@@ -26,6 +26,9 @@ Following are the supported element types in StableHLO:
     (represents a pair of `f64`). Exact representation of complex types
     (e.g. whether the real part or the imaginary part comes first in memory)
     is implementation-defined.
+  * **String type**  represent a sequence of bytes. Exact representation of
+    string type (e.g. null terminated or not, encoding etc.) is
+    implementation-defined.
```

Comment:
Proposed rewording:

```
**String type** represent a sequence of bytes and is referred to in the document 
as `string`. Exact representation of string type (e.g. null terminated or not,
encoding etc.) is implementation-defined.

Strings types are not first class, i.e. StableHLO doesn't support values of
string types. Some StableHLO ops can take strings as inputs, but they are never
produced as outputs.
```

Feedback:
  1. Let's move out of element types and all the way to the very bottom of the section.
  2. Also let's update the third bullet in function types to say something like "Input types and output types are one of `tensor`, `token` or `tuple`.".